_docs/install/kubernetes.md

```diff
@@ -7,10 +7,7 @@ module: install/kubernetes
 
 # Kubernetes
 
->**Note**: Full Kubernetes integration has been submitted for inclusion in Kubernetes 1.7.  You can
-track progress at [https://github.com/kubernetes/kubernetes/pull/42156](https://github.com/kubernetes/kubernetes/pull/42156).  Once approved and merged
-into Kubernetes, StorageOS will be available as a storage provider in the base Kubernetes distribution.  Until then,
-you must re-compile Kubernetes if you wish to test StorageOS integration with Kubernetes.
+>**Note**: Native support for StorageOS volumes has been added in Kubernetes 1.7, which is the minimum requirement.
```

Comment:
Can we just say that the minimum requirement is Kubernetes 1.7 in the Prerequisites section?